### PR TITLE
(DOCSP-8927,8932): Port: Hosting Metadata

### PR DESCRIPTION
**Pull request by nlarew**
## Jira

- [(DOCSP-8927): Port: Configure File Metadata](https://jira.mongodb.org/browse/DOCSP-8927)
- [(DOCSP-8932): Port: File Metadata Attributes](https://jira.mongodb.org/browse/DOCSP-8932)

## Staged Changes

- [Configure File Metadata](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker/hosting-metadata/hosting/configure-file-metadata)
- [File Metadata Attributes](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker/hosting-metadata/hosting/file-metadata-attributes)

### CODE REVIEW

File: source/hosting/configure-file-metadata.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+You can configure a file's :doc:`metadata attributes
```
Comment (MongoCaleb) [inline source/hosting/configure-file-metadata.txt]: Do we want any sort of "why" here -- why would I want to do this? It seems obvious to me, but will it to our customers?

File: source/includes/steps-configure-file-metadata-import-export.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+     realm-cli export --app-id=<App ID>
+---
+title: Add Attributes to the Metadata Configuration File
+ref: add-attributes-to-the-metadata-configuration-file
+content: |
+  The metadata for all hosted files associated with your application is
+  configured by an array of metadata attribute definition objects that
+  you define in a file named :ref:`metadata.json
+  <hosting-metadata-json>` in your application's ``hosting``
+  subdirectory.
```
Comment (MongoCaleb) [inline source/includes/steps-configure-file-metadata-import-export.yaml]: I think this is my brain, not yours...but the staging environment displays this as "RefRole" and is not a link.

File: source/includes/steps-configure-file-metadata-import-export.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+  Realm will automatically infer and assign a ``Content-Type`` to every
+  file you upload. If you you want to override the default
+  ``Content-Type`` or specify additional :doc:`metadata attributes
+  </hosting/file-metadata-attributes>` for a file, add an entry to
+  :ref:`metadata.json <hosting-metadata-json>` for the file and
```
Comment (MongoCaleb) [inline source/includes/steps-configure-file-metadata-import-export.yaml]: Same thing here. So maybe it's not me?

File: source/includes/steps-configure-file-metadata-import-export.yaml (hunk 2 above)
Comment (nlarew) [reply source/includes/steps-configure-file-metadata-import-export.yaml]: Yeah NextGen doesn't render refs atm :(

File: source/includes/steps-configure-file-metadata-import-export.yaml (hunk 3 above)
Comment (nlarew) [reply source/includes/steps-configure-file-metadata-import-export.yaml]: Yeah NextGen doesn't render refs atm :(

File: source/hosting/configure-file-metadata.txt (hunk 1 above)
Comment (nlarew) [reply source/hosting/configure-file-metadata.txt]: Yup good call, could definitely use some up front context here

**Pull request merged**